contribution/words.json

```diff
@@ -503,10 +518,12 @@
 		"Want some?",
 		"You got the hands, I got the job to fit them",
 		"you need some BTC?",
+		"You look like you could use a job"
```

Comment:
If you decide to reopen this or another PR. Here's the formatting error. A comma is missing at the end.